modules/class.js

```diff
@@ -0,0 +1,97 @@
+import '../src/style.css';
+import { form, input, list } from './DOMelements.js';
+
+class ToDo {
+  constructor() {
+    this.tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  }
+
+  addTask(taskDescription) {
+    const newTask = {
+      description: taskDescription,
+      completed: false,
+      index: this.tasks.length + 1,
+    };
```

Comment:
Great job @nxn404-n 🤝

- [ ] As stated by the previous reviewer, kindly make sure your indexes start from 1 instead of 0.
![index](https://user-images.githubusercontent.com/63917725/220920264-017b5b64-1186-46e0-af9f-3044bbe47916.PNG)